SETUP.md

```diff
@@ -86,6 +88,10 @@ git config core.eol lf
 git config core.filemode false
 ```
 
+### ARM-bsed CPUs
+
+Tests that require the use of ChromeDriver (both Karma and Dusk tests) will not work inside Docker when running on ARM-based CPUs (e.g. Macs running Apple Silicon). In this scenario, these tests should be run outside of a container.
```

Comment:
the tests should still be able to be run inside container by running chromedriver locally and specifying `DUSK_WEBDRIVER_URL` and `APP_URL` (to something accessible from local) inside `.env.dusk.local`.

or alternatively maybe use a dedicated docker instance running chromedriver (it'll also allow the custom chromium settings for php instance to be removed).

Reply:
This doesn't help with running karma - karma's chrome launcher doesn't support remote and needs to use `karma-webdriver-launcher` which doesn't target chromedriver but selenium server instead, which doesn't support running in docker on m1

Reply:
Hm, karma supports system installed chromium by specifying `CHROME_BIN`

Reply:
which isn't useful for remote driver?

Reply:
yeah but at least the non-snap native chromium method would work

Reply:
also I just noticed "ARM-bsed" 🙃

Reply:
may want to update the wording to say Chrome instead of ChromeDriver as the latter isn't used by karma (and using remote chromedriver may work for dusk?)